includes/class-move.php

```diff
@@ -69,13 +32,22 @@ public static function account( $from, $to ) {
 		}
 
 		// Update the movedTo property.
-		\update_user_option( $user->get__id(), 'activitypub_move_to', $to );
+		if ( $user->get__id() > 0 ) {
```

Comment:
This is very nit-picky, so feel free to ignore: We could check explicitly for `$user->get__id() === Actors::BLOG_USER_ID` instead.